lib/daru/maths/statistics/vector.rb

```diff
@@ -7,6 +7,8 @@ module Statistics
       module Vector
         def mean
           @data.mean
+          rescue ZeroDivisionError
```

Comment:
An btw that looks like a bug. `sum` is supposed to skip over all missing elements in the `missing_values`.